pkgs/applications/editors/emacs-modes/emacspeak/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub, rsync, gnused, emacs, tcl, tclx, espeak-ng }:
+
+stdenv.mkDerivation rec {
+  pname = "emacspeak";
+  version = "51.0";
+
+  src = fetchFromGitHub {
+    owner = "tvraman";
+    repo = "emacspeak";
+    rev = version;
+    sha256 = "06azq2566pj4pnyawqm9387ybp3wr1pr6828hhdxhki4dpn9pj8g";
+  };
+
+  nativeBuildInputs = [ rsync gnused ];
+  buildInputs = [ emacs tcl tclx espeak-ng ];
+  postPatch = ''
+    substituteInPlace servers/native-espeak/Makefile --replace '/usr/include/tcl$(TCL_VERSION)' ${tcl}/include
+    substituteInPlace servers/native-espeak/Makefile --replace 'LIBPARENTDIR = ''${PREFIX}/'  'LIBPARENTDIR = /'
+  '';
+
+  preConfigure = ''
+    make config
+  '';
+
+  postBuild = ''
+    make espeak
```

Comment:
makeFlags won't be effective here, but `make espeak` just calls `servers/native-espeak/Makefile`. So you can also do this:

```
make -C servers/native-espeak "TCL_INCLUDE=${tcl}/include" "LIBPARENTDIR = /"
```